src/panoramic/cli/local/get.py

```diff
@@ -1,16 +1,24 @@
 from typing import Optional
 
 from panoramic.cli.local.reader import FileReader
-from panoramic.cli.pano_model import PanoModel, PanoVirtualDataSource
+from panoramic.cli.pano_model import PanoField, PanoModel, PanoVirtualDataSource
 from panoramic.cli.state import VirtualState
 
 
 def get_state(target_dataset: Optional[str] = None) -> VirtualState:
-    """Build a representation of what VDS and models are on local filesystem.
+    """
+    Build a representation of what VDS and models are on local filesystem.
     """
     packages = FileReader().get_packages()
     data_sources = []
     models = []
+    fields = []
+
+    if target_dataset is None:
+        for field, path in FileReader().get_global_fields():
```

Comment:
Can we use one reader for both `get_packages` and `get_global_fields` ?

Reply:
Fixed in 28fa8e9f243a1f098bc26a2b952039de6079b8c6.